pkg/cloudprovider/providers/azure/azure_backoff.go

```diff
@@ -321,8 +321,12 @@ func (az *Cloud) DeleteRouteWithRetry(routeName string) error {
 
 // CreateOrUpdateVMWithRetry invokes az.VirtualMachinesClient.CreateOrUpdate with exponential backoff retry
 func (az *Cloud) CreateOrUpdateVMWithRetry(vmName string, newVM compute.VirtualMachine) error {
```

Comment:
would you remove this func `CreateOrUpdateVMWithRetry` as it's not invoked by any code